app/views/layouts/_header.html.erb

```diff
@@ -17,9 +17,10 @@
     <div class="collapse navbar-collapse" id="header-navbar-collapse">
       <form class="form-inline mr-3" id="searchform" autocomplete="off">
         <div class="input-group">
+          <label id="searchLabel" for="searchform_input">
```

Comment:
The label tag is empty and not closed?

Reply:
Hey @VladimirMikulic The reason I added this without any text like this is because I didn't want to change the page content while at the same time add a form-label. And the reason I added this is because some old screen readers don't recognize the aria assistive apis :sweat_smile: Thanks a lot for the review :tada:

Reply:
Not closing tag is a bad practice. It can cause structural and styling issues down the road.
In my opinion, `aria-label` is the way to go. Messing with the DOM can be dangerous.
We can't forever be backwards compatible :)

Reply:
Hey @VladimirMikulic That's right :sweat_smile: I will update this

Reply:
No problem. Take your time.